binaries/benchmark_helper.cc

```diff
@@ -376,6 +377,35 @@ void writeOutput(
   }
 }
 
+void logBenchmarkResult(
+    const std::string& type,
+    const std::string& metric,
+    const std::string& unit,
+    const int value) {
+  LOG(INFO) << caffe2::NetObserverReporterPrint::IDENTIFIER << "{"
+            << "\"type\": \"" << type << "\", "
+            << "\"metric\": \"" << metric << "\", "
+            << "\"unit\": \"" << unit << "\", "
+            << "\"value\": " << c10::to_string(value) << "}\n";
+}
+
+long getVirtualMemoryIfOptionEnabled(bool FLAGS_measure_memory) {
```

Comment:
This function doesn't have an implementation for Windows. Would you please add them into your code?
There are two functions that may suit your needs:
https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/heapwalk?view=vs-2019
https://docs.microsoft.com/zh-cn/windows/win32/psapi/collecting-memory-usage-information-for-a-process

Reply:
Thanks for the pointer, will work on it!